client/dashboard/sites/overview/site-action-menu.tsx

```diff
@@ -0,0 +1,51 @@
+import { useQuery } from '@tanstack/react-query';
+import { DropdownMenu, MenuGroup, MenuItem } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { moreVertical } from '@wordpress/icons';
+import { addQueryArgs } from '@wordpress/url';
+import { useAnalytics } from '../../app/analytics';
+import { isFSEActiveQuery } from '../../app/queries/site-themes';
+import { getSiteEditUrl } from '../../utils/site-url';
+import type { Site } from '../../data/types';
+
+const SiteActionMenu = ( { site }: { site: Site } ) => {
+	const { recordTracksEvent } = useAnalytics();
+	const { data: isFSEActive, isLoading: isFSEActiveLoading } = useQuery(
+		isFSEActiveQuery( site.ID )
+	);
+
+	const trackActionClick = ( action: string ) => {
+		recordTracksEvent( 'calypso_sites_dashboard_site_action_menu_click', { action } );
+	};
+
+	const handleEditSite = () => {
+		trackActionClick( 'edit_site' );
```

Comment:
Nit: I think we need to be consistent with kebab vs snake case here... in upsell tracking we use kebab (`agency-site-share`) but here we're using snake case